src/datamigration/azext_datamigration/generated/_help.py

```diff
@@ -0,0 +1,423 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['datamigration'] = '''
+    type: group
+    short-summary: Manage Data Migration
+'''
+
+helps['datamigration to-sql-managed-instance'] = """
+    type: group
+    short-summary: Manage database migrations to SQL Managed Instance.
```

Comment:
Could you remove `to-` in command group name?
```suggestion
helps['datamigration sql-managed-instance'] = """
    type: group
    short-summary: Manage database migrations to SQL Managed Instance.
```